### PR TITLE
[Starter Packs] Posts tab

### PR DESCRIPTION
**Pull request by gaearon**
Yay or nay?

https://github.com/bluesky-social/social-app/assets/810438/1e93a4be-b46c-4b48-b0c0-01da7ba7098b

### CODE REVIEW

File: src/screens/StarterPack/StarterPackScreen.tsx [hunk 1]
```diff
@@ -206,6 +210,18 @@ function StarterPackScreenInner({
                 />
               )
             : null}
+          {showPostsTab != null
```
Comment (gaearon) [inline src/screens/StarterPack/StarterPackScreen.tsx:213]: oops should be just boolean

**Review by estrattonbailey — APPROVED** [review]
NICE, just tested on web and works great, feeds tab or no

**Review by pfrazee — APPROVED** [review]
Tested on Web and iOS, works well

I'm +1 I think

**Comment by estrattonbailey** [comment]
Tested query invalidation using iOS too e.g. remove all feeds and it removes the tab.

Yeah I'm +1, nice little preview of what you get.

**Pull request merged**